packages/tezos-embedded-protocol-005-PsBabyM1/tezos-embedded-protocol-005-PsBabyM1.7.0/opam

```diff
@@ -0,0 +1,35 @@
+opam-version: "2.0"
+maintainer: "contact@tezos.com"
+authors: [ "Tezos devteam" ]
+homepage: "https://www.tezos.com/"
+bug-reports: "https://gitlab.com/tezos/tezos/issues"
+dev-repo: "git+https://gitlab.com/tezos/tezos.git"
+license: "MIT"
+depends: [
+  "tezos-tooling" { with-test & = version }
+  "dune" { >= "1.11" }
+  "tezos-protocol-005-PsBabyM1" { = version }
+  "tezos-protocol-updater" { = version }
+]
+build: [
+  [ "rm" "src/proto_005_PsBabyM1/lib_protocol/dune" ]
+  [
+    "%{tezos-protocol-compiler:lib}%/replace"
+    "%{tezos-protocol-compiler:lib}%/dune_protocol.template"
+    "%{build}%/src/proto_005_PsBabyM1/lib_protocol/dune"
+    "%{tezos-protocol-compiler:lib}%/final_protocol_versions"
+    "005_PsBabyM1"
```

Comment:
This command (re)generates the dune file from its template.
- Actually, once you've generated the dune file a first time (and a dune file is included in the sources), you can already ask dune to regenerate it using `dune build @$src/proto_005_PsBabyM1/lib_protocol/runtest_dune_template --auto-promote`. 
- There's no reason to regenerate the dune file while building the opam package, the one already there should be just fine (and running the test will check that!). I've removed this step for all the other "protocols" but I left it once here as a """reference""".

Reply:
That makes sense! Yes, dune will error out with a clear error message if the promotion does alter the checked in version.

Reply:
The other good side effect of removing this will be this camelus warning will disappear:
> tezos-embedded-protocol-005-PsBabyM1.7.0 has some warnings:
>
> warning 41: Some packages are mentioned in package scripts or features, but there is no dependency or depopt  toward them: "tezos-protocol-compiler"

The promotion rules above reference `%{tezos-protocol-compiler:...}` which is not in the `depends` list for the package.

Reply:
Removed